tests/load/config.js

```diff
@@ -0,0 +1,15 @@
+import crypto from "k6/crypto";
+
+export const URL = "http://107.22.129.12";
+export const CART_PATH = "/boltpay/cartDataLoadTest";
+export const SHIPPING_PATH = "/boltpay/shipping";
+export const PREAUTH_PATH = "/boltpay/api/create_order";
+const SIGNING_SECRET = "f527a8972d13c1ecdc31fd45a47054a249fe058b5bede0fc652dec986c95bfea";
```

Comment:
Consider adding a comment explaining this hard coded values  At minimum, can we specify that it is a place holder that needs to be replaced?

Reply:
Sounds good, I added the comments